README.md

```diff
@@ -1,13 +1,11 @@
 # Cluedo - Mystery Randomizer
+This is my second week into learning js, this project was supposed to teach me more about loors and arrays. 
+I tried to fucus on learning more than the design, will make tome to come back for that. Boot-camp life is busy! 
```

Comment:
My personal opinion is focus on the main topic for the week.  Functionality is beauty this week!